code_generator_raw_uproot/transformer_capabilities.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "Uproot transformer using native uproot arguments",
+  "description": "Extracts data from flat ntuple style root files.",
+  "limitations": "Would be good to note what isn't implemented",
+  "file-formats": ["parquet"],
```

Comment:
Need to indicate the native ability to write root files too. See example in [python code generator](https://github.com/ssl-hep/ServiceX/blob/d92749f04d35035c51ccf4900e9ee1122daa6e96/code_generator_python/transformer_capabilities.json#L5)

Reply:
Done